packages/roosterjs-editor-dom/lib/htmlSanitizer/getAllowedValues.ts

```diff
@@ -1,14 +1,134 @@
 import { cloneObject } from './cloneObject';
 import { PredefinedCssMap, StringMap, StyleCallbackMap } from 'roosterjs-editor-types';
 
-const ALLOWED_HTML_TAGS = (
-    'BODY,H1,H2,H3,H4,H5,H6,FORM,P,BR,NOBR,HR,ACRONYM,ABBR,ADDRESS,B,' +
-    'BDI,BDO,BIG,BLOCKQUOTE,CENTER,CITE,CODE,DEL,DFN,EM,FONT,I,INS,KBD,MARK,METER,PRE,PROGRESS,' +
-    'Q,RP,RT,RUBY,S,SAMP,SMALL,STRIKE,STRONG,SUB,SUP,TEMPLATE,TIME,TT,U,VAR,WBR,XMP,INPUT,TEXTAREA,' +
-    'BUTTON,SELECT,OPTGROUP,OPTION,LABEL,FIELDSET,LEGEND,DATALIST,OUTPUT,IMG,MAP,AREA,CANVAS,FIGCAPTION,' +
-    'FIGURE,PICTURE,A,NAV,UL,OL,LI,DIR,UL,DL,DT,DD,MENU,MENUITEM,TABLE,CAPTION,TH,TR,TD,THEAD,TBODY,' +
-    'TFOOT,COL,COLGROUP,DIV,SPAN,HEADER,FOOTER,MAIN,SECTION,ARTICLE,ASIDE,DETAILS,DIALOG,SUMMARY,DATA'
-).split(',');
+const ALLOWED_HTML_TAG_MAP = {
+    // Allowed tags
+    a: true,
+    abbr: true,
+    address: true,
+    area: true,
+    article: true,
+    aside: true,
+    b: true,
+    bdi: true,
+    bdo: true,
+    blockquote: true,
+    body: true,
+    br: true,
+    button: true,
+    canvas: true,
+    caption: true,
+    center: true,
+    cite: true,
+    code: true,
+    col: true,
+    colgroup: true,
+    data: true,
+    datalist: true,
+    dd: true,
+    del: true,
+    details: true,
+    dfn: true,
+    dialog: true,
+    dir: true,
+    div: true,
+    dl: true,
+    dt: true,
+    em: true,
+    fieldset: true,
+    figcaption: true,
+    figure: true,
+    font: true,
+    footer: true,
+    form: true,
+    h1: true,
+    h2: true,
+    h3: true,
+    h4: true,
+    h5: true,
+    h6: true,
+    head: true,
+    header: true,
+    hgroup: true,
+    hr: true,
+    html: true,
+    i: true,
+    img: true,
+    input: true,
+    ins: true,
+    kbd: true,
+    label: true,
+    legend: true,
+    li: true,
+    main: true,
+    map: true,
+    mark: true,
+    menu: true,
+    menuitem: true,
+    meter: true,
+    nav: true,
+    ol: true,
+    optgroup: true,
+    option: true,
+    output: true,
+    p: true,
+    picture: true,
+    pre: true,
+    progress: true,
+    q: true,
+    rp: true,
+    rt: true,
+    ruby: true,
+    s: true,
+    samp: true,
+    section: true,
+    select: true,
+    small: true,
+    span: true,
+    strike: true,
+    strong: true,
+    sub: true,
+    summary: true,
+    sup: true,
+    table: true,
```

Comment:
What about svg tag?

Reply:
SVG tags will be treated as unknown tags and replaced with SPAN.
Before this change, all SVG tags will be removed.
Let's see if users are satisfied with the new behavior.